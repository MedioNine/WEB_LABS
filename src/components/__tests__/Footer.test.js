import React from 'react';
import ReactTestUtils from 'react-dom/test-utils'; 
import Footer from '../FooterComponent';
import { shallow } from 'enzyme';


describe('FooterComponent', () => {

    

      describe('Header container initial', () => {
        const FooterContainer = (<Footer/>)

        it('renders properly', () => {
            expect(FooterContainer).toMatchSnapshot()
          })    
        
          const component = shallow(<Footer/>)

          it('company name visible', () => {
            expect(component.find('span').text()).toEqual(' Wanderlace')
          })
          
          

     })

     

})