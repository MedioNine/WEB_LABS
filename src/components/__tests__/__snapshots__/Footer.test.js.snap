// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent Header container initial renders properly 1`] = `<Footer />`;
